*** waf-1.6.4/waflib/Tools/compiler_fc.py.orig	Sun Apr 17 06:02:48 2011
--- waf-1.6.4/waflib/Tools/compiler_fc.py	Thu May  5 11:12:04 2011
***************
*** 6,16 ****
  from waflib.Tools import fc
  
  fc_compiler = {
  	'win32'  : ['gfortran','ifort'],
  	'darwin' : ['gfortran', 'g95', 'ifort'],
! 	'linux'  : ['gfortran', 'g95', 'ifort'],
  	'java'   : ['gfortran', 'g95', 'ifort'],
  	'default': ['gfortran'],
  	'aix'    : ['gfortran']
  }
  
--- 6,16 ----
  from waflib.Tools import fc
  
  fc_compiler = {
  	'win32'  : ['gfortran','ifort'],
  	'darwin' : ['gfortran', 'g95', 'ifort'],
! 	'linux'  : ['gfortran', 'g95', 'ifort', 'g77' ],
  	'java'   : ['gfortran', 'g95', 'ifort'],
  	'default': ['gfortran'],
  	'aix'    : ['gfortran']
  }
  
